articles/quickstart/native/react-native/00-login.md

```diff
@@ -46,9 +46,21 @@ You can find this at the top of your `AndroidManifest.xml` file located in the `
 
 <%= include('../../../_includes/_logout_url') %>
 
-::: note
-If you are following along with the sample project you downloaded from the top of this page, the logout URL you need to whitelist in the Allowed Logout URLs field is the same as the callback URL.
-:::
+#### iOS logout URL
+
+```text
+{PRODUCT_BUNDLE_IDENTIFIER}://${account.namespace}/ios/{PRODUCT_BUNDLE_IDENTIFIER}/callback
+```
+
+Remember to replace `PRODUCT_BUNDLE_IDENTIFIER` with your actual application's bundle identifier name.
```

Comment:
I think you should include the `{` in the replace instructions or omit it in the URL as right now they could easily end up leaving the `{` and `}` in the URLs.